core/src/test/java/fr/sncf/osrd/train/TestTrains.java

```diff
@@ -107,7 +109,8 @@ private static Map<String, RollingStock.ModeEffortCurves> createModeEffortCurves
                 RJSLoadingGaugeType.G1,
                 linearModeEffortCurves,
                 "thermal",
-                "1"
+                "1",
+                new String[]{"BAL","BAPR", "TVM300", "TVM430"}
```

Comment:
```suggestion
                new String[]{"BAL", "BAPR", "TVM300", "TVM430"}
```

Checkstyle error, happens in other places in this file as well

Reply:
done